Makefile

```diff
@@ -239,25 +232,6 @@ helm-docs: helm-docs-plugin ## Generates markdown documentation for helm charts
 
 ##@ Deployment
 
-ifndef ignore-not-found
-  ignore-not-found = false
-endif
-
-.PHONY: kind-create-cluster
-kind-create-cluster: kind ## Create a kind cluster for integration tests.
-	if ! $(KIND) get clusters 2>/dev/null | grep -q "^$(KIND_CLUSTER_NAME)$$"; then \
-		kind create cluster --name $(KIND_CLUSTER_NAME) --config $(KIND_CONFIG_FILE) --kubeconfig $(KIND_KUBE_CONFIG); \
-	fi
-
-.PHONY: kind-load-image
-kind-load-image: kind-create-cluster docker-build ## Load the image into the kind cluster.
-	$(KIND) load docker-image --name $(KIND_CLUSTER_NAME) $(IMAGE)
-
-.PHONY: kind-delete-custer
-kind-delete-custer: kind ## Delete the created kind cluster.
-	$(KIND) delete cluster --name $(KIND_CLUSTER_NAME) && \
-	rm -f $(KIND_KUBE_CONFIG)
-
```

Comment:
The kind related targets are used in local tests if the developer does not have a k8s cluster already, perhaps we can keep these targets.

Reply:
Fair point, I've personally used `k3s`/`k3d` and `minikube` before as well but it looks like `kind` is fairly common among open source projects